test/TestProjects/FirstTest-Cadl/Generated/Demo2Client.cs

```diff
@@ -84,39 +84,54 @@ public Demo2Client(TokenCredential credential, Uri endpoint, CadlfirsttestClient
         }
 
         /// <summary> Return hi again. </summary>
+        /// <param name="p2"> The String to use. </param>
+        /// <param name="p1"> The String to use. </param>
         /// <param name="action"> The RoundTripModel to use. </param>
+        /// <param name="contentType"> The String to use. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="action"/> is null. </exception>
-        public virtual async Task<Response<Thing>> HelloAgainAsync(RoundTripModel action, CancellationToken cancellationToken = default)
+        /// <exception cref="ArgumentNullException"> <paramref name="p2"/>, <paramref name="p1"/>, <paramref name="action"/> or <paramref name="contentType"/> is null. </exception>
+        public virtual async Task<Response<Thing>> HelloAgainAsync(string p2, string p1, RoundTripModel action, string contentType, CancellationToken cancellationToken = default)
```

Comment:
If there is only one value for content type, we should hard code it instead of exposing a parameter. See example https://github.com/pshao25/autorest.csharp/blob/0ea9774167b6e88bfeb539cbfc1cfa0a794721e3/test/TestServerProjects/media_types/Generated/MediaTypesClient.cs#L193

Reply:
I think if the content-type is one value and it is 'application/json' we may hard code in Create***Request; if multiple-value, will add contentType parameter